<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pay</class>
 <widget class="QDialog" name="Pay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Pay{
background-color: #2a6496;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame {
    background-color: #fff;
    border: none;
    border-radius: 5px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Price</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="paylineEdit"/>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>pay</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="payydateEdit"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Parent Phone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="parentphonelineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
